packages/contracts/contracts/registry/InterfaceBasedRegistry.sol

```diff
@@ -11,7 +11,7 @@ import "../core/component/DaoAuthorizableUpgradeable.sol";
 /// @author Aragon Association - 2022
 /// @notice An [ERC-165](https://eips.ethereum.org/EIPS/eip-165)-based registry for contracts
 //TODO Make this PluginUUPSUpgradeable
-abstract contract InterfaceBasedRegistry is DaoAuthorizableUpgradeable, UUPSUpgradeable {
+abstract contract InterfaceBasedRegistry is UUPSUpgradeable, DaoAuthorizableUpgradeable {
```

Comment:
IMO, this should be a `PluginUUPSUpgradeable` too being installed on our managing Framework DAO. However, this clearly goes beyond this PR, but deserves a discussion.